src/api/telemetry/TelemetryAPI.js

```diff
@@ -397,8 +446,34 @@ define([
      * @method limitEvaluator
      * @memberof module:openmct.TelemetryAPI~TelemetryProvider#
      */
-    TelemetryAPI.prototype.limitEvaluator = function () {
-        return this.legacyProvider.limitEvaluator.apply(this.legacyProvider, arguments);
+    TelemetryAPI.prototype.limitEvaluator = function (domainObject) {
+        return this.getLimitEvaluator(domainObject);
+    };
+
+    /**
+     * Get a limit evaluator for this domain object.
+     * Limit Evaluators help you evaluate limit and alarm status of individual
+     * telemetry datums for display purposes without having to interact directly
+     * with the Limit API.
+     *
+     * This method is optional.
+     * If a provider does not implement this method, it is presumed
+     * that no limits are defined for this domain object's telemetry.
+     *
+     * @param {module:openmct.DomainObject} domainObject the domain
+     *        object for which to evaluate limits
+     * @returns {module:openmct.TelemetryAPI~LimitEvaluator}
+     * @method limitEvaluator
+     * @memberof module:openmct.TelemetryAPI~TelemetryProvider#
+     */
+    TelemetryAPI.prototype.getLimitEvaluator = function (domainObject) {
+        var provider = this.findLimitEvaluator(domainObject);
+        if (!provider) {
+            return {
+                evaluate: function () {}
```

Comment:
See previous comment about a default limit evaluator

Reply:
See [comment](https://github.com/nasa/openmct/pull/1941#pullrequestreview-105464455) above, not going to add a default at this time due to this being in flux.